src/services/publishing/publishing.service.ts

```diff
@@ -66,6 +66,13 @@ export class PublishingService {
         return;
       }
 
+      if (!video.size) {
+        this.#logger.error(
+          'Videos are not being populated with a size field at time of publishing!',
+        );
+        video.size = 0;
```

Comment:
Don't we have access to the raw bytes of the video? Can't we just pull that file size?

Reply:
This only handles publishing to hive in a cron setting

Reply:
So the video size can be automatically set during the upload step maybe?

Reply:
I believe it is or will be. The size property is optional because the video document isn't created with a size. This conditional is mostly just for keeping typescript happy.

Reply:
Shouldn't it stop trying to publish the video tho if the video size isn't set? We don't want to post bad data on the user's behalf